pydra/engine/helpers.py

```diff
@@ -128,10 +128,10 @@ def save(task_path: Path, result=None, task=None, name_prefix=None):
         if result:
             if task_path.name.startswith("Workflow"):
                 # copy files to the workflow directory
-                if result.output is None:
-                    raise ValueError(
-                        "Workflow output cannot be None, define output(s) with set_output"
-                    )
+                # if result.output is None:
```

Comment:
remove any code that is not required instead of commenting it out. you can always go back to history to retrieve it.

Reply:
Sorry forgot to delete!  Will be more careful next time